src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -26,6 +28,7 @@ class TaskBuilderTest extends UnitTest {
 
   import mesosphere.mesos.protos.Implicits._
 
+  implicit val clock = Clock.systemUTC()
```

Comment:
any particular reason for using the system clock vs. a fake one?

Reply:
I think this happened during my merge conflict resolving, good catch! Changed this to the new `SettableClock`.